.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Don't commit the `.vs` folder to git. Feel free to add it to the .gitignore file if that makes it easier for you.